zipline/test_algorithms.py

```diff
@@ -947,6 +949,28 @@ def handle_data(self, data):
         self.i += 1
 
 
+class TestLiquidationAlgorithm(TradingAlgorithm):
```

Comment:
We're generally moving away from the model of subclassing `TradingAlgorithm` for specific algos.  I'd rather write this as:

```
def initialize(context):
    ...

def handle_data(context, data):
    ...

algo = TradingAlgorithm(initialize=initialize, handle_data=handle_data, ...)
```